packages/csharp/PortingAssistant/PortingAssistant.Api/Application.cs

```diff
@@ -132,18 +132,11 @@ public void SetupConnection(bool console = false)
             _connection.On<string, bool>("checkInternetAccess", request =>
             {
                 var httpService = _services.GetRequiredService<IHttpService>();
-                try
```

Comment:
I get that in some failure cases, the IsCompletedSuccessfully is false anyway, but aren't there other possible ways this could throw an exception? It looks fine to me, but I'm just wondering what the reason for this change was in the first place and if it's safer to keep the try catch.

Reply:
After investigating further, I think you're right @Beau-Gosse-dev, we still need the try catch because the Download method can/will throw an exception if the request comes back unsuccessful. But the try catch needs to be around each call so that one package being removed from the data store won't cause the method to fail. I have a follow-up commit coming soon.